code/index.html

```diff
@@ -2,21 +2,169 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <title>The Clickbait: &#128240</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <link href="https://fonts.googleapis.com/css?family=Alata&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
```

Comment:
Tip! You can actually select multiple fonts in Google Fonts. Then it will give you only one `<link>` to insert here. Of course this works as well, but one less row of code is great!